.travis.yml

```diff
@@ -1,10 +1,28 @@
 language: node_js
 node_js:
   - "8"
-  - "9"
 
 cache: yarn
 
 install: yarn
+
+env:
+  global:
+    - DOCKER_COMPOSE_VERSION=1.23.2
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+before_script:
+  - sleep 3
+  - docker-compose up -d --build
+
 script:
   - yarn lint
+  - yarn ci
+
+after_script:
+  - docker-compose down
```

Comment:
We might need to add
```
services:
  - docker
```
to this file?

Reply:
I'm not really sure, it runs the test 👍

Reply:
Yeah, maybe adding this just gives you access to the docker command and as we don't use that it doesn't really matter